cmd/server/assets/apikeys/os.html

```diff
@@ -0,0 +1,76 @@
+{{$authApp := .authApp}}
+
+  <div class="form-group">
+    <select class="form-control{{if $authApp.ErrorsFor "os"}} is-invalid{{end}}" name="os" id="os" onchange="osSelect()">
+      <option value="0" {{if (eq $authApp.OS .unknownOS)}}selected{{else}}disabled{{end}}>Select OS...</option>
+      <option value="{{.iOS}}" {{if (eq $authApp.OS .iOS)}}selected{{end}}>iOS</option>
+      <option value="{{.android}}" {{if (eq $authApp.OS .android)}}selected{{end}}>Android</option>
+      <option value="{{.bothOS}}" {{if (eq $authApp.OS .bothOS)}}selected{{end}}>iOS and Android</option>
+    </select>
+    {{if $authApp.ErrorsFor "os"}}
+    <div class="invalid-feedback">
+      {{joinStrings ($authApp.ErrorsFor "os") ", "}}
+    </div>
+    {{end}}
+  </div>
+
+  <div class="form-label-group" id="iosDIV">
+    <input type="text" id="iosAppID" name="iosAppID" class="form-control{{if $authApp.ErrorsFor "iosAppID"}} is-invalid{{end}}" value="{{$authApp.IOSAppID}}" placeholder="Application ID" autofocus>
+    <label for="iosAppID">iOS Application ID</label>
```

Comment:
nit: Similarly to below: if "Application ID" here means "bundle identifier", it might be good to call it that in UI.

Reply:
I'll update the UI to move to bundle id for android.